gh-pages/docs/community/projects.md

```diff
@@ -1,45 +1,84 @@
-Katzner Lab, Ludwig-Maximilians-Universität München <br />
-Moser Group, Kavli Institute for Systems Neuroscience <br />
-Mackenzie Mathis Lab, EPFL <br />
-Anne Churchland Lab, UCLA <br />
-Hui-Chen Lu Lab, University of Indiana <br />
-Karl Deisseroth Lab, Stanford University <br />
-Harvey Lab, Harvard Medical School <br />
-Datta Lab, Harvard Medical School <br />
-Sabatini Lab, Harvard Medical School <br />
-Smirnakis Lab, Harvard Medical School <br />
-Joshua Gordon Lab, National Institute of Neurological Disorders <br />
-Wang Lab, Duke University <br />
-Angelaki Lab, New York University <br />
-McGinley Lab, Baylor College of Medicine <br />
-Jerry Chen Lab, Boston University <br />
-Seung Lab, Princeton University <br />
-Siapas Lab, California Institute of Technology <br />
-Engel Lab, Cold Spring Harbor Laboratory <br />
-Tuthill Lab, University of Washington <br />
-Applied Physics Lab, Johns Hopkins University <br />
-Loren Frank Lab, University of California, San Francisco <br />
-Rick Gerkin, Arizona State University <br />
-… 40+ other labs
+DataJoint was originally developed by working systems neuroscientists at Andreas Tolias’ Lab at Baylor College of Medicine to meet the needs of their own research.
+
+Below is a partial list of known DataJoint users. Please let us know if you would like to add another group or make a correction by <a href="https://github.com/datajoint/datajoint-elements/issues">submitting a GitHub issue</a>.
+
+Please visit our <a href="https://catalog.datajoint.io/">Catalog</a> for a more detailed discussion of past projects.
+
+<h3>Multi-lab collaboratives</h3>
+
+<ul>
+    <li><a href="https://www.internationalbrainlab.com/#home">International Brain Lab</a></li>
+    <li><a href="https://www.simonsfoundation.org/funded-project/multi-regional-neuronal-dynamics-of-memory-guided-flexible-behavior/">Mesoscale Activity Project</a></li>
+    <li><a href="https://www.iarpa.gov/research-programs/microns">MICrONS</a></li>
+    <li>Sainsbury Wellcome Centre <a href="https://www.sainsburywellcome.org/web/">"Aeon"</a></li>
+    <li>U19 Awards
+        <ul>
+        <li>NYU <a href="https://www.osmonauts.org/">"Osmonauts"</a></li>
+        <li>Harvard <a href="https://www.teamdope.org/">"Dope"</a></li>
+        <li>Columbia <a href="https://confluence.columbia.edu/confluence/display/zmbbi/U19+Data+Science">"MoC3"</a></li>
+        <li>Princeton <a href="https://www.braincogs.org/">"BrainCoGs"</a></li>
+        <li>Rochester <a href="https://reporter.nih.gov/project-details/10047607">Neural basis of causal inference</a></li>
+        </ul>
+    </li>
+</ul>
+
+
+<h3>Invidiual Labs</h3>
+
+<ul>
+    <li>Allen Institute
+        <ul>
+        <li><a href="https://alleninstitute.org/what-we-do/brain-science/research/mindscope-program/">Mindscope</a></li>
+        <li><a href="https://alleninstitute.org/what-we-do/brain-science/research/allen-institute-neural-dynamics/">Neural Dynamics</a></li>
```

Comment:
```suggestion
```

Reply:
This is currently an embedded list:
> - Allen Institute
>    - Mindscope
>    - Neural Dynamics

Your edit would remove the last line. Should we not mention the neural dynamics project?